wp-lazy-loading.php

```diff
@@ -94,38 +94,32 @@ function _wp_lazy_loading_add_attribute_to_attachment_image( $attr ) {
 // The following functions will be merged to core.
 
 /**
- * Get the HTML tags to lazy-load.
+ * Determine whether to add the `loading` attribute to the specified tag in the specified context.
  *
  * @since (TBD)
  *
- * @return array List of tags to add loading="lazy" attributes to.
+ * @param string  $tag_name The tag name.
+ * @param string  $context Additional context, like the current filter name or the function name from where this was called.
+ * @return boolean Whether to add the attribute.
  */
-function wp_get_lazy_load_tags() {
-	// See https://github.com/whatwg/html/issues/2806.
-	$supported_tags = array( 'img', 'iframe' );
+function wp_add_lazy_loading_to( $tag_name, $context ) {
```

Comment:
I think this name is misleading. This function returns a value, it is not used to add something. The filter can be used to modify the result (and in a way, in fact add lazy-loading support), but the function name itself is still not correct, especially when looking at where it is called.

How about `wp(_is)_lazy_loading_enabled( $tag_name, $context )` or `wp_requires_lazy_loading( $tag_name, $context )`?

Reply:
Yeah, I see your point. Perhaps `wp_lazy_loading_enabled( $tag_name, $context )`? Thinking that sounds best "self-documenting" wise :)

Updating.

Reply:
SGTM